Automattic/woocommerce-payments
# Disable unavailable payment methods due to country restrictions

Fixes #3638

#### Changes proposed in this Pull Request
Hide payment methods that are not available due to Stripe country restrictions. Like ACH that's only available on US or BECS on AU.

h/t @tpaksu for the suggestion of using payment method fees to check if it's available. I thought of adding the countries list on the client, but doesn't make sense to duplicate it, if we can do the same with the fees that already check the country on the server.

I opted to directly hide the payment method from the list. But we could also disable it and show a pill informing the customer that it isn't available in their country. Something like:
<img width="703" alt="Screenshot 2022-01-14 at 12 14 07" src="https://user-images.githubusercontent.com/7670276/149506677-fc3e8207-ded7-429b-aed6-3f052f88b2bf.png">


#### Testing instructions
- You need to disable additional payments on the server in order to test it while there are no single country payment methods. Comment your Stripe account country on the countries constant in `Fee Service`
- Go to **Payments > Settings > Payment methods**
- Unavailable payment methods should be hidden now.

-------------------
